packages/lib/types.ts

```diff
@@ -13,3 +15,13 @@ export enum ApplicationType {
 	Mobile = 2,
 	Cli = 3,
 }
+
+export type FetchBlobOptions = {
+	path?: string;
+	method?: string;
+	maxRedirects?: number;
+	timeout?: number;
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
+	headers?: any;
```

Comment:
May as well fix the type? Isn't it just `Record<string, string>`?

Reply:
You are right, I should be more careful. My bad.